lesson16/index.js

```diff
@@ -61,23 +268,43 @@ function createTabContents() {
 function appendFragment(values) {
     const fragment = document.createDocumentFragment();
 
-    //記事タイトルの数だけliとaを作成
     let articleTitles = [];
     values.forEach((e) => {
         articleTitles.push(e.title);
     });
 
+    let articleComments = [];
+    values.forEach((e) => {
+        articleComments.push(e.comments);
+    });
+
+    //記事タイトルの数だけliを追加
     for (let i = 0; i < articleTitles.length; i++) {
         const li = document.createElement("li");
         const a = document.createElement("a");
+        const commentIconWrapper = document.createElement("div");
+        const commentIcon = document.createElement("img");
+        const commentLength = document.createElement("div");
+        const numberOfComments = articleComments[i].length;
 
-        li.classList.add("tab__contents-item");
+        li.classList.add("tab__contents-item","js-tabContentsItem");
         a.classList.add("tab__contents-link");
         a.href = "#";
         a.insertAdjacentHTML("beforeend", articleTitles[i]);
+        commentIcon.src = "./img/icon-comment.svg";
+        commentIconWrapper.classList.add("tab__contents-icon");
+        commentLength.classList.add("tab__contents-info");
 
         li.appendChild(a);
         fragment.appendChild(li);
```

Comment:
```suggestion
        fragment.appendChild(li).appendChild(a);
```

nits
You can also write something like this

Reply:
I realize now that I've been writing in a complicated way ...! 
Thank you very much.
6407a72